src/studio/src/repositories/docker-compose.yml

```diff
@@ -19,6 +19,9 @@ services:
     ports:
       - "3000:3000"
       - "222:22"
+    environment:
```

Comment:
I think these docker environment specific docker-compose files could be deleted.. @lorang92 agree? I don't see when they are ever used.

Reply:
I guess this might have been used for local testing?